CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 - **[Breaking]** Try and As conversion helpers are removed in favor of using
   other cast libraries.
 - **[Breaking]** Removed `Value.IsDefault` method.
+- **[Breaking]** Removed Load* functions.
+- **[Breaking]** Removed NewYAMLProviderFromReader* functions.
```

Comment:
Consider changing `Removed` to `Unexport` for the `NewYAMLProviderFromReader*` functions.